external/build_all.sh

```diff
@@ -16,6 +16,16 @@ set -o pipefail
 
 source $(dirname "$0")/common_functions.sh
 
+if [ -n "${EXTERNAL_AQA_IMAGE}" ] ; then\
+  echo "EXTERNAL_AQA_IMAGE is decalred as $EXTERNAL_AQA_IMAGE; it would break this script. Unset it."
+  exit 1
+fi
+
+if [ -z "${version}" ] ; then\
+  echo "'version' (of jdk, eg 17) is mandatory variable for this script to run"
+  exit 1
+fi
+
 # Cleanup any old containers and images
```

Comment:
Please see https://github.com/adoptium/aqa-tests/issues/5657 ; I think that script should lbe dropped. It was broken since February,  when the version was dropped from loops but its usage was left on place. Its slightly earlier in this PR, without reply.

Reply:
I hope that file will be removed. If not, then authors of commit  which made the build-all.sh should elaborate on correct usage.